packages/react-data-grid/src/RowGroup.tsx

```diff
@@ -42,7 +42,7 @@ const RowGroup = forwardRef<HTMLDivElement, Props>(function RowGroup(props, ref)
   }
 
   function onClick() {
-    props.eventBus.dispatch(EventTypes.SELECT_CELL, { rowIdx: props.idx });
+    props.eventBus.dispatch(EventTypes.SELECT_CELL, { rowIdx: props.idx, idx: 0 });
```

Comment:
I've set it to `idx: 0`, I don't think we have a live demo of this component in our examples, but I rendered it by tweaking the Canvas, and it works okay.
Since this renders just one grid-wide "cell", I don't think it can have any other idx.